scripts/Logistics-Pipes.zs

```diff
@@ -0,0 +1,873 @@
+recipes.remove(<LogisticsPipes:item.PipeFluidRequestLogistics>);
+
+// --- Logistics Fluid Satellite Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidSatellite>);
+
+// --- Logistics Fluid Insertion Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidInsertion>);
+
+// --- Logistics Fluid Extractor Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidExtractor>);
+
+
+// ||||| Chassis |||||
+
+
+// --- Logistics Chassi MK1
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk1>);
+
+// --- Logistics Chassi MK2
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk2>);
+
+// --- Logistics Chassi MK3
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk3>);
+
+// --- Logistics Chassi MK4
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk4>);
+
+// --- Logistics Chassi MK5
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk5>);
+
+
+
+// *======= Adding Back Recipes =======*
+// --- Assembler Recipes --- 
+
+// - LV Tier
+// ||||| Blocks |||||
+// --- Power Junction
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 1152, 800, 30);
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.tin> * 576, 800, 30);
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.solderingalloy> * 288, 800, 30);
+
+// --- Unrouted Logistics Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeItemsBasicTransport> * 2, [<ore:stickLongSteel> * 2, <bartworks:BWPumpParts>, <ore:wireFineGold> * 2, <gregtech:gt.integrated_circuit:18> * 0], null, 100, 30);
+Assembler.addRecipe(<LogisticsPipes:item.PipeItemsBasicTransport> * 8, [<ore:stickLongSteel> * 2, <gregtech:gt.metaitem.03:32023> * 2, <ore:wireFineGold> * 8, <gregtech:gt.integrated_circuit:18> * 0], null, 100, 30);
+Assembler.addRecipe(<LogisticsPipes:item.PipeItemsBasicTransport> * 16, [<ore:stickLongStainlessSteel> * 2, <ore:plateBorosilicateGlass> * 8, <ore:wireFineGold> * 16, <gregtech:gt.integrated_circuit:18> * 0], null, 100, 30);
+
+
+// --- Basic Logistics Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeItemsBasicLogistics> * 4, [<LogisticsPipes:item.PipeItemsBasicTransport> * 4, <ore:gearGtSmallGold> * 2, <ore:stickDiamond> * 2, <ore:circuitBasic> * 2, <ore:wireFineSilver> * 16, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.solderingalloy> * 72, 800, 30);

```

Comment:
Diamond components take forever to make at low voltage tier. Best avoid it. Probably could just add an alternative recipe for each of these, that instead of using 2 diamond rods, use 2?4? diamond. This way you loss some material efficiency but can speed up a lot.

Reply:
Alternatively, add a recipe to craft diamond rods from diamonds with files.

Reply:
Added a recipe with 4 diamonds = 2 pipes